ports/servoshell/desktop/minibrowser.rs

```diff
@@ -282,6 +270,17 @@ impl Minibrowser {
                     let Some(servo_fbo) = servo_framebuffer_id else {
                         return;
                     };
+
+                    if let Some(status_text) = &self.status_text {
+                        egui::containers::popup::show_tooltip_at(
+                            ctx,
+                            LayerId::new(Order::Tooltip, "tooltip layer".into()),
+                            ui.id(),
```

Comment:
I believe this is passing the parent's ID as the ID of this widget, whereas we'll need to make a unique ID here. The original code did this using `tooltip_for_status_text.into()` which seems correct to me.